app/vmagent/remotewrite/remotewrite.go

```diff
@@ -80,9 +82,13 @@ var (
 		`For example, if m{k1="v1",k2="v2"} may be sent as m{k2="v2",k1="v1"}`+
 		`Enabled sorting for labels can slow down ingestion performance a bit`)
 	maxHourlySeries = flag.Int("remoteWrite.maxHourlySeries", 0, "The maximum number of unique series vmagent can send to remote storage systems during the last hour. "+
-		"Excess series are logged and dropped. This can be useful for limiting series cardinality. See https://docs.victoriametrics.com/victoriametrics/vmagent/#cardinality-limiter")
+		"Excess series are logged and dropped. This can be useful for limiting series cardinality. "+
+		"Setting this flag to '-1' sets limit to maximum possible value which is useful in order to enable series tracking without enforcing limits. "+
+		"See https://docs.victoriametrics.com/victoriametrics/vmagent/#cardinality-limiter")
 	maxDailySeries = flag.Int("remoteWrite.maxDailySeries", 0, "The maximum number of unique series vmagent can send to remote storage systems during the last 24 hours. "+
-		"Excess series are logged and dropped. This can be useful for limiting series churn rate. See https://docs.victoriametrics.com/victoriametrics/vmagent/#cardinality-limiter")
+		"Excess series are logged and dropped. This can be useful for limiting series churn rate. "+
+		"Setting this flag to '-1' sets limit to maximum possible value which is useful in order to enable series tracking without enforcing limits. "+
```

Comment:
Should this behavior be consistent for all cardinality limiters?

 Also, it'll bring constant memory footprint and increase vmagent memory usage.